integration/tests/addons/addons_test.go

```diff
@@ -75,6 +76,11 @@ var _ = Describe("(Integration) [EKS Addons test]", func() {
 				WithStdin(bytes.NewReader(data))
 			Expect(cmd).To(RunSuccessfully())
 
+			rawClient = getRawClient(clusterName)
+			serverVersion, err := rawClient.ServerVersion()
+			Expect(err).ToNot(HaveOccurred())
+			Expect(serverVersion).To(ContainSubstring(api.LatestVersion))
+
```

Comment:
adding a *new* integration test felt like extreme overkill, so I picked a test that already created a cluster and had it use `latest` :shrug: doesn't feel super clean, but better than adding a new suite